organization/philanthropy.md

```diff
@@ -0,0 +1,34 @@
+---
+type: resource
+explains: our policy on philanthropy
+---
+
+# Philanthropy policy
+
+The Foundation for Public Code is currently wholly supported by philanthropists who share our commitment to [our mission](https://about.publiccode.net/organization/mission.html). All donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/public_benefit_organisations) (ANBI).
```

Comment:
I suggest a return and putting the `ANBI` in the link as it's part of the same entity:

```suggestion
The Foundation for Public Code is currently wholly supported by philanthropists who share our commitment to [our mission](https://about.publiccode.net/organization/mission.html).
All donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization (ANBI)](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/public_benefit_organisations).
```